met-api/src/met_api/services/comment_service.py

```diff
@@ -107,3 +110,23 @@ def extract_comments(cls, survey_submission: SubmissionSchema, survey: SurveySch
         comments = [cls.__form_comment(key, submission.get(key, ''), survey_submission, survey)
                     for key in text_component_keys if submission.get(key, '') != '']
         return comments
+
+    @classmethod
+    def extract_comments_to_spread_sheet(cls, survey_id):
+        """Extract comments to spread sheet."""
+        comments = Comment.get_comments_by_survey_id(survey_id)
+        formatted_comments = [
+            {
+                'commentNumber': comment.id,
+                'dateSubmitted': str(comment.submission_date),
+                'author': f'{comment.first_name} {comment.last_name}',
```

Comment:
We only capture the staff users name not the public, so this will always be empty, we might need remove this column, another thing to talk to MA/Carly

Reply:
Good point, noted